webapp/src/components/form/MarkDownForm.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import MarkdownIt from 'markdown-it'
+import MdEditor from 'react-markdown-editor-lite'
+
+import 'react-markdown-editor-lite/lib/index.css';
+
+
+const mdParser = new MarkdownIt();
+
+class MarkDownEditor extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            inputValue: ""
+        }
+    }
+    
+
+ handleEditorChange({html, text}) {    
+  this.setState({
+    inputValue: text
+  })
+
+  this.props.onChange({html, text})
```

Comment:
Rather send just the text to the onChange function, which I assume is a string(?), then you don't need any special logic to handle validation of objects in answer.value in Application.js line 333. Also our backend expects answer.value to be a single string anyway, not an object.